ci-operator/step-registry/cucushift/hypershift-extended/cilium/cucushift-hypershift-extended-cilium-commands.sh

```
@@ -25,74 +28,31 @@ if [[ -f "${SHARED_DIR}/kubeconfig.kubeadmin" ]]; then
+# where <version> is a tag from https://quay.io/repository/cilium-charts-dev/cilium
+cilium install \
+    --repository oci://quay.io/cilium-charts-dev/cilium \
+    --version 1.20.0-dev-dev.86-8df528194f0b \
+    --set debug.enabled=true \
+    --set k8s.requireIPv4PodCIDR=true \
+    --set logSystemLoad=true \
+    --set ipv6.enabled=false \
+    --set identityChangeGracePeriod=0s \
+    --set ipam.mode=cluster-pool \
+    --set "ipam.operator.clusterPoolIPv4PodCIDRList={${PODCIDR}}" \
+    --set ipam.operator.clusterPoolIPv4MaskSize=${HOSTPREFIX} \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)